### DISCUSSION TITLE
Inquiry Regarding Favicon Color Change

### DISCUSSION THREAD

**Discussion by FREDVUNI** [community/community#113246] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm aware of the white favicon issue or discussion on [GitHub](https://github.com/orgs/community/discussions/12744). My question is, why has it changed? It was black yesterday. Is there a specific reason for this?

![image](https://github.com/community/community/assets/41730664/9bec5f83-a483-4b6b-945e-5b17fe200484)

**Comment by akshaypandey89** [comment, marked as answer]
use systems default dark mode rather than using chrome or applications dark mode.
![Screenshot 2024-03-19 123252](https://github.com/community/community/assets/152717788/ada12941-539c-4ac3-ada9-96d618ebf74b)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FREDVUNI ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.

**Reply by FREDVUNI** [reply]
Thank you @queenofcorgis
